### PR TITLE
various improvements

### PR DESCRIPTION
**Pull request by modularTaco**
- fixed the labels for the grafana configmap, as it was using the labels from grafana operator and not from grafana configmap

- Allow the grafana resources (operator and configmap) to be installed in a different namespace than s3-proxy itself.  
  Common setup is that the grafana sidecar only looks for dashboard in grafanas namespace (probably `monitoring`)
  
- Allow secrets to be defined outside of the helm charts values

- bumped app version to 3.2.0, adjusted old app version in readme

### CODE REVIEW

**Comment by oxyno-zeta** [comment]
Hello @modularTaco ! 

Very good PR and work ! Thanks ! Can you do a little change please ? Can you change the version to 2.2.0 ? I prefer to not increase from 2.1.0 to 2.3.1 without any intermediate version. 

Thanks !

File: stable/s3-proxy/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "3.1.0"
+appVersion: "3.2.0"
 description: A Helm chart for Kubernetes to deploy S3-Proxy
 name: s3-proxy
-version: 2.1.0
+version: 2.3.1
```
Comment (oxyno-zeta) [inline stable/s3-proxy/Chart.yaml]: Can you change the version to 2.2.0 please ?

**Review by oxyno-zeta — CHANGES_REQUESTED** [review]

**Comment by modularTaco** [comment]
> Hello @modularTaco !
> 
> Very good PR and work ! Thanks ! Can you do a little change please ? Can you change the version to 2.2.0 ? I prefer to not increase from 2.1.0 to 2.3.1 without any intermediate version.
> 
> Thanks !

You're absolutely right :+1:

File: stable/s3-proxy/Chart.yaml (hunk 1 above)
Comment (modularTaco) [reply stable/s3-proxy/Chart.yaml]: done :+1:

**Review by oxyno-zeta — APPROVED** [review]

**Comment by oxyno-zeta** [comment]
This is merged ! Thanks for your work.

**Pull request merged**